tests/testthat/test-include-hba1c.R

```diff
@@ -0,0 +1,64 @@
+lab_forsker <- tibble::tribble(
+  ~patient_cpr, ~samplingdate, ~analysiscode, ~value,
+  "498718589803", "20220101", "NPU27300", 49,
+  "498718589804", "20220101", "NPU27300", 47,
+  # Duplicate patient_cpr but with the old units.
+  "498718589803", "20220101", "NPU03835", 6.5,
+  "498718589805", "20220101", "NPU03835", 6.5,
+  "498718589806", "20220101", "NPU03835", 6.3,
+  "498718589807", "20220101", "NPU00000", 100,
+  "498718589808", "20220101", "NPU00000", 5
+)
+
+expected <- tibble::tribble(
+  ~pnr, ~include_hba1c,
+  "498718589803", TRUE,
```

Comment:
Don't have duplicates, keep the two earliest dates of being true, drop all later dates.